packages/bappo-components/src/components/IconCard/IconCard.tsx

```diff
@@ -73,15 +57,15 @@ const StyledView = styled(View)`
   padding: 8px;
   border-radius: 3px;
   width: 100%;
-  height: ${(props) => `${props.$size}px`};
-  background: ${(props) => props.$color || '#D8D8D8'};
+  height: ${({ $size }) => `${$size}px`};
+  background: ${({ $color }) => $color || '#D8D8D8'};
   display: flex;
   align-items: center;
   justify-content: center;
   margin: 8px;
 `;
 
-const StyledIcon = styled(Icon)`
-  width: ${(props) => `${(props.$size * 2) / 5}px` || '16px'};
-  font-size: ${(props) => `${props.$size / 3}px` || '14px'};
+const StyledIcon = styled(Icon)<{ $size: any }>`
+  width: ${({ $size }) => `${($size * 2) / 5}px` || '16px'};
```

Comment:
Similar issue as above

Reply:
Thanks for the suggestions! I've committed a changes for this.